core/ChangeLog.md

```diff
@@ -9,20 +9,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 _Most likely to be released on July 1st._
 
 ### Added
-- Soft lock.  #958
-- Auto lock.  #1027
+- Soft lock.  [#958]
```

Comment:
I prefer the in-line markdown syntax for hyperlinks. This has an advantage the user does not have to scroll the long document back-and-forth in case it is being viewed/edited in plaintext form.

```suggestion
- Soft lock.  [#958](https://github.com/trezor/trezor-firmware/issues/958)
```

Reply:
My rationale for leaving the links out is that this makes the changelog easier to copy-paste, and makes it less spammy (all the issue links look the same; if you are viewing in plaintext, you can just copy&paste the number into github).

This would not apply to other links, just to issue links.

Reply:
ACK